recipe/meta.yaml

```diff
@@ -1,10 +1,10 @@
 package:
     name: gridutils
-    version: 1.01.0
+    version: v1.0
 
 source:
     git_url: https://github.com/sakov/gridutils-c.git
-    git_tag: master  # upstream does not create releases :-(
```

Comment:
Maybe just ask for tags? Otherwise, we should be pinning at (approximately) immutable things like SHAs/tags, rather than "master".